lib/omniauth/strategies/openid_connect.rb

```diff
@@ -14,6 +14,8 @@ class OpenIDConnect
       include OmniAuth::Strategy
       extend Forwardable
 
+      SUPPORTED_GRANT_TYPES = %w[code id_token token].freeze
```

Comment:
why did you add

```ruby
SUPPORTED_GRANT_TYPES = %w[code id_token token]
```

if 

> currently, only `:code` (Authorization Code grant) and `:id_token` (Implicit grant) are valid

?

Reply:
I'm going to remove the 'token' response type (together with the combined id_token + token), it's out of scope of my current work and adds too much complexity. I'll update this soon.